### ISSUE TITLE
Accessibility for Collapsible Menu View Ellipsis

### ISSUE THREAD

**Issue by PseudoFaux** [cerner/terra-core#1151]
# Issue Description

For the collapsible menu view, when items are rolled up into the ellipsis, the ellipsis button itself does not have an aria-label. With this in mind, a screenreader when clicking/tabbing the ellipsis, the screenreader does not read anything.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior

When using a screenreader and tabbing over to the '...' iconOnly button, it should have an aria-label that has some method of letting the user know that there was a '...' button highlighted. Perhaps "More"/"Ellipsis"/etc.

## Current Behavior

When tabbing over to a rolled up Collapsible Menu, the ellipsis button reads nothing when using a screenreader.

## Environment

* Component Version: 1.8.0
* Browser Name and Version: Chrome 63 (and likely others)
* Operating System and version (desktop or mobile): OSX/Windows 10, Desktop
